manifests/m/Microsoft/NetMon/3.4/Microsoft.NetMon.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.NetMon
+PackageVersion: 3.4
+PackageLocale: en-US
+Publisher: Microsoft
```

```suggestion
Publisher: Microsoft Corporation
```
To match Registry / Control Panel